From 25a7ee2a4c5f8d357a8771e17007c3d9861ce7bc Mon Sep 17 00:00:00 2001
From: Yaroslav Halchenko <debian@onerussian.com>
Date: Tue, 23 Jun 2020 18:45:45 -0400
Subject: [PATCH] BF: annotate tests in test_utils_cached_dataset with
 @skip_if_no_network

Since they all rely on access to a github repository ATM.
See https://github.com/datalad/datalad/issues/4653 for a possible RF to avoid that
---
 datalad/tests/test_utils_cached_dataset.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/datalad/tests/test_utils_cached_dataset.py b/datalad/tests/test_utils_cached_dataset.py
index f3db768fc..8df3f3718 100644
--- a/datalad/tests/test_utils_cached_dataset.py
+++ b/datalad/tests/test_utils_cached_dataset.py
@@ -25,6 +25,7 @@ from datalad.tests.utils import (
     assert_raises,
     assert_result_count,
     assert_true,
+    skip_if_no_network,
     with_tempfile
 )
 from unittest.mock import patch
@@ -34,6 +35,7 @@ CACHE_PATCH_STR = "datalad.tests.utils_cached_dataset.DATALAD_TESTS_CACHE"
 CLONE_PATCH_STR = "datalad.tests.utils_cached_dataset.Clone.__call__"
 
 
+@skip_if_no_network
 @with_tempfile(mkdir=True)
 def test_get_cached_dataset(cache_dir):
 
@@ -150,6 +152,7 @@ def test_get_cached_dataset(cache_dir):
             assert_is(ds, ds2)
 
 
+@skip_if_no_network
 @with_tempfile(mkdir=True)
 def test_cached_dataset(cache_dir):
 
@@ -250,6 +253,7 @@ def test_cached_dataset(cache_dir):
         assert_not_equal(first_repopath, second_repopath)
 
 
+@skip_if_no_network
 @with_tempfile(mkdir=True)
 def test_cached_url(cache_dir):
 
-- 
2.24.0

